ui/templates/base.html

```diff
@@ -11,7 +11,10 @@
     {% load raven %}
     <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
     <link rel="stylesheet" type="text/css" href="{% static 'hijack/hijack-styles.css' %}" />
-    <script type="text/javascript">
+      {% if video_annotations %}
+        <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.1/jquery.min.js"></script>
```

Comment:
I'm guessing this means that this `videojs-annotation-comments` plugin requires jQuery? Could you not add this dependency as a `require` call in `static/js/lib/video.js`?